src/pages/Defi/views/StakingVaults.tsx

```diff
@@ -12,9 +16,50 @@ const DefiHeader = () => {
   )
 }
 
+const FoxFarmCTA = () => {
+  const ethLogoSrc = 'https://assets.coincap.io/assets/icons/256/eth.png'
+  const foxLogoSrc = 'https://assets.coincap.io/assets/icons/256/fox.png'
```

Comment:
We should use smaller icon, grep for `2x.png` in the codebase or have a look in `src/test/constants`
```suggestion
  const foxLogoSrc = 'https://assets.coincap.io/assets/icons/fox@2x.png'
```